start-console.sh

```diff
@@ -6,6 +6,8 @@ CONSOLE_IMAGE=${CONSOLE_IMAGE:="quay.io/openshift/origin-console:latest"}
 CONSOLE_PORT=${CONSOLE_PORT:=9000}
 CONSOLE_IMAGE_PLATFORM=${CONSOLE_IMAGE_PLATFORM:="linux/amd64"}
 
+PLUGIN_NAME=$(node -r ts-node/register -e "console.log(JSON.stringify(require('./plugin-metadata.ts')))" | jq -r .default.name)
```

Comment:
Does this introduce a new dependency on `jq` that we didn't have before? Does this work?

```suggestion
PLUGIN_NAME=$(node -r ts-node/register -e "console.log(require('./plugin-metadata.ts').default.name)"
```

Also will this pick up the ts-node from node_modules or will it need to be installed globally on the system?